nullaway/src/test/java/com/uber/nullaway/NullAwayFrameworkTests.java

```diff
@@ -365,4 +365,75 @@ public void systemConsoleNullable() {
             "}")
         .doTest();
   }
+
+  @Test
+  public void mapGetOrDefault() {
```

Comment:
If it's important we are testing here without `AcknowledgeRestrictiveAnnotations` and that's the point of the test case and the reason we aren't using `defaultCompilationHelper`, then, shouldn't that be reflected in the name?

Reply:
Applied your suggestion to fix